product/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+
+from .views import ProductListView, FilterView
+
+urlpatterns = [
+    path('/allitem-list', ProductListView.as_view()),
+    path('/allitem-filter', FilterView.as_view())
```

Comment:
엔드포인트 주소 확인 부탁 드립니다. allitem-list 일단 좀 어색보입니다.
엔드포인트 주소는 - 로 구분지을 수 있는데 all-item-list 로 해도 되지만 product 로 라우팅 되므로 아래와 같이 주소 지정해도 될것 같습니다. 참고하셔서 명확한 주소로 정해주세요!
```suggestion
    path('', ProductListView.as_view()),
    path('/filter', FilterView.as_view())
```

Reply:
바꾸었습니다